src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java

```diff
@@ -283,5 +283,5 @@ private Object convertArgument(Object jsonArgument, String type) throws BadInstr
     private static final List<Pathfinder.PathElement> NEVER_MATCH_PATH = Collections.<Pathfinder.PathElement>emptyList();
 
     @SuppressWarnings("unused")
-    private static final String LOGTAG = "MixpanelAPI.EditProtocol";
+    private static final String LOGTAG = "MixpanelAPI.EProtocol";
```

Comment:
It turns out there is a 23 character limit on logtags we've been blowing past (I've actually filed this as #176) - since at this point all of our logtags are prefixed with "MixpanelAPI" (which is super useful!) that leaves us with a bunch of abbrevs. and short msgs.